docs/java-rest/high-level/index.asciidoc

```diff
@@ -6,33 +6,11 @@
 [partintro]
 --
 
-deprecated[7.15.0, The High Level REST Client is deprecated in favour of the {java-api-client}/index.html[Java API Client].]
+removed[8.0.0, The High Level REST Client has been removed in favour of the {java-api-client}/index.html[Java API Client].]
```

Comment:
This page is the only non remove diff of this PR.
Should we add more information here? Or should this page be completely removed as well?